src/constants.ts

```diff
@@ -18,7 +18,7 @@ const localize: nls.LocalizeFunc = nls.config({
 
 export const CONFIG = {
   SHOW_DEPENDENCY_INSTALL: "deviceSimulatorExpress.showDependencyInstall",
-  SHOW_NEW_FILE_POPUP: "pacifica.showNewFilePopup"
+  SHOW_NEW_FILE_POPUP: "deviceSimulatorExpress.showNewFilePopup"
```

Comment:
I'm not super familiar with this part. Are you familiar with whether this needs to be changed in [package.json](https://github.com/microsoft/vscode-python-devicesimulator/blob/users/t-vali/spelling-errors/package.json#L177) as well?

Reply:
Good catch, updated in next commit